package.json

```diff
@@ -31,7 +31,7 @@
     "wait-namespace": "node ./scripts/wait-on-temporal.mjs",
     "lint": "eslint packages/*/src --ext .ts --no-error-on-unmatched-pattern && prettier --end-of-line auto --check .",
     "lint.prune": "ts-prune --error -p tsconfig.prune.json --ignore \"used in module\" --skip \".d.ts\"",
-    "format": "prettier --write .",
+    "format": "eslint packages/*/src --ext .ts --no-error-on-unmatched-pattern --fix && prettier --write .",
```

Comment:
nit: not related to formatting

Reply:
maybe this?

https://github.com/temporalio/sdk-typescript/pull/965/commits/6e02630cb50f97d0f77661327969cc3f91cd91d7

looking for one command to run locally before committing